CHANGELOG.md

```diff
@@ -7,7 +7,8 @@
 - add landing page https://mandiant.github.io/capa/ @williballenthin #2310
 - add rules website https://mandiant.github.io/capa/rules @DeeyaSingh #2310
 - add .justfile @williballenthin #2325
- 
+- support analyzing BinExport2 files generated by Ghidra #1950 @williballenthin @mehunhoff @mr-tz
```

Comment:
In theory, this implementation should not be specific to Ghidra, In practice, the underlying tools (IDA, Ghidra, Binary Ninja, etc.) that generate BinExport files do not generate the files in the same way e.g https://github.com/google/binexport/issues/126. This is why we narrowed our focus to Ghidra and I only mention Ghidra in the CHANGELOG because I can't be confident that BinExport files generated by the other tools will work without additional testing and workarounds.